olm_deploy/manifests/deployment.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metering-ansible-operator-registry
```

Comment:
I realize we may want to change this to also include the `df.NamespacePrefix` field as a prefix to the `metadata.name` to support multi-tenancy.

Reply:
ah, yeah, I think that is what I am hitting on with my parallelization work. I see errors like:
```
            	Error:      	Received unexpected error:
            	            	failed to install metering: failed to create the metering resources: failed to create the metering role: the namespace of the provided object does not match the namespace sent on the request
            	Test:       	TestManualMeteringInstall/ValidHDFS-ReportStaticInputData
```

Reply:
In that case, it's most likely we're passing the `deploy.config.Namespace` as the namespace parameter to the Create method, which is different than what's in the `metadata.namespace` of the resource we're also passing as we're decoding a YAML manifest to a Go object, and modifying the resource from there.